script/script.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable linebreak-style */
+const navWrapper = document.querySelector('.hamburger-menu');
+const hamMenu = document.querySelectorAll('.line');
+const sideNav = document.querySelector('.side-nav');
+const sideLinks = document.querySelectorAll('.links');
+const Toggler = (e) => {
+  e.preventDefault();
+  hamMenu.forEach((a) => a.classList.toggle('active'));
+  sideNav.classList.toggle('hide');
+};
+
+navWrapper.addEventListener('click', Toggler);
+
+// eslint-disable-next-line no-plusplus
```

Comment:
[Optional]

- [ ] Linters are made to force us to follow certain guidelines and work according to a specific style. It's better if you avoid disabling it and instead work on solving the errors 👍